numismatic/feeds/base.py

```diff
@@ -241,15 +238,56 @@ def _make_request(self, api_url, params=None, headers=None, raw=False):
             data = response
         return data
 
-
 @attr.s
 class WebsocketClient(abc.ABC):
     '''Base class for WebsocketClient feeds'''
     # FIXME: Is this really an ABC? What abstractmethods are there?
 
     exchange = None
     websocket_url = None
-    websocket = attr.ib(default=None)
+    _websocket_map = dict()
```

Comment:
perhaps this should be called `websocket_connection_pool`

Reply:
No, map is better. Pool would imply that their substitutable which isn't the case.